WaterJug.py

```diff
@@ -4,101 +4,102 @@
 
 class WaterJug:
 
-    def __init__(self,fileptr):
-        self.num=int(fileptr.readline())
+    def __init__(self,fileptr): #constructor
+        self.num=int(fileptr.readline()) #number of glass
```

Comment:
i'll try change it. if you want to merge it i might get more motivated to do so :)